__tests__/main.test.ts

```diff
@@ -66,6 +67,21 @@ describe('options', () => {
     });
   });
 
+  describe('getDist', () => {
+    afterEach(() => {
+      delete process.env['INPUT_DIST'];
+    });
+
+    test('should return undefined when dist is omitted', async () => {
+      expect(getDist()).toBeUndefined();
+    });
+
+    test('should return a string when dist is provided', () => {
+      process.env['INPUT_DIST'] = 'foo-dist';
```

Comment:
Why the usage of this env? I don't see it used in the code.

Reply:
the github action runner creates all inputs as env vars, prefixed with `INPUT_`, so `INPUT_DIST` is equivalent to `${ inputs.dist }` in the action itself, or `with: dist: 'foo-dist'` via a workflow.

> When you specify an input in a workflow file or use a default input value, GitHub creates an environment variable for the input with the name INPUT_<VARIABLE_NAME>. The environment variable created converts input names to uppercase letters and replaces spaces with _ characters.
[Source](https://docs.github.com/en/actions/creating-actions/metadata-syntax-for-github-actions#inputs)

This is how the Sentry Action tests are implemented, as it's (presumably) easier to unit test by setting env vars, than by running the workflow with different sets of `with:` params. I just followed the format used for the existing tests rather than implementing any new testing approach.